README.md

```diff
@@ -39,13 +39,13 @@ Input:
 Output:
 ```ts
 export interface ExampleSchema {
-  firstName: string;
-  lastName: string;
+  "firstName": string;
```

Comment:
It seems like better code style to me. Eg. I use a lint rule to prevent unnecessary quoting of object keys.

If you want, feel free to rebase, I'll merge it, and can then enhance/test the encoding behavior on my own time.